install/operator/deploy/role.yaml

```diff
@@ -1,58 +1,206 @@
-apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
this changed from `rbac.authorization.k8s.io/v1` to `rbac.authorization.k8s.io/v1beta1`, should it have?

Reply:
same as the next comment